pjlib/include/pj/ssl_sock.h

```diff
@@ -668,6 +668,20 @@ typedef struct pj_ssl_sock_cb
      */
     pj_bool_t (*on_connect_complete)(pj_ssl_sock_t *ssock,
 				     pj_status_t status);
+    
+    /**
+     * This callback is called when certificate verification is being done.
+     * Certification info can be obtained from #pj_ssl_sock_info.
```

Comment:
Add info that this callback is currently only implemented for OpenSSL backend.
Also below for transport_tls